modules/creating-a-workbench-select-ide.adoc

```diff
@@ -0,0 +1,26 @@
+:_module-type: CONCEPT
+
+[id="creating-a-workbench-select-ide_{context}"]
+= Creating a workbench and selecting an IDE
+
+[role="_abstract"]
+A workbench is an isolated area where you can examine and work with ML models. You can also work with data and run programs, for example to prepare and clean data. While a workbench is not required if, for example, you want to use {productname-short} to only service a model that you already have, a workbench is needed for most data science workflow tasks, such as writing code to process data or training a model.
```

Comment:
- Add comma after first 'for example'.
- Could cut the longer 3rd sentence down a bit:

'While a workbench is not required if, for example, you only want to service an existing model, one is needed for most data science workflow tasks, such as writing code to process data, or training a model.'